### PR TITLE
Create adding-fonts-to-the-wsk.md

### PR DESCRIPTION
**Pull request by bugrakocak**

### CODE REVIEW

File: checklists/adding-fonts-to-the-wsk.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Adding Fonts to the WSK
+category: Front-End
+date: "2020-03-20"
+tags: ['font', 'scss']
+description: Here is the checklist that we follow to import fonts to our projects.
+---
+
+- [ ] Get the fonts!  
+Search through the web but if you can't manage to find kindly ask your designer to provide you.
```
Comment (alpcanaydin) [inline checklists/adding-fonts-to-the-wsk.md]: Maybe we can mention this to-do for free license fonts.

File: checklists/adding-fonts-to-the-wsk.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Adding Fonts to the WSK
+category: Front-End
+date: "2020-03-20"
+tags: ['font', 'scss']
+description: Here is the checklist that we follow to import fonts to our projects.
+---
+
+- [ ] Get the fonts!
```
Comment (akemalakcay) [inline checklists/adding-fonts-to-the-wsk.md]:
```suggestion
- [ ] Get the fonts  
```

File: checklists/adding-fonts-to-the-wsk.md (hunk 1 above)
Comment (akemalakcay) [inline checklists/adding-fonts-to-the-wsk.md]:
```suggestion
If the project's font is free licence you can either download it from internet or have the designer to provide it for you.
```

File: checklists/adding-fonts-to-the-wsk.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+date: "2020-03-20"
+tags: ['font', 'scss']
+description: Here is the checklist that we follow to import fonts to our projects.
+---
+
+- [ ] Get the fonts!  
+Search through the web but if you can't manage to find kindly ask your designer to provide you.
+
+- [ ] Convert your fonts  
+We use fonts as 'woff', 'woff2' and 'ttf' formats in our projects so you need to convert your fonts into those. You can use 'fontplop' desktop app for it.
```
Comment (akemalakcay) [inline checklists/adding-fonts-to-the-wsk.md]: I don't know how you guys are proceeding if the font is in .otf format?

File: checklists/adding-fonts-to-the-wsk.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: Adding Fonts to the WSK
+category: Front-End
+date: "2020-03-20"
+tags: ['font', 'scss']
```
Comment (akemalakcay) [inline checklists/adding-fonts-to-the-wsk.md]:
```suggestion
tags: ['font', 'scss', 'wsk']
```

File: checklists/adding-fonts-to-the-wsk.md (hunk 3 above)
Comment (bugrakocak) [reply checklists/adding-fonts-to-the-wsk.md]: Fontplop accepts both ttf and otf formats.

**Pull request merged**